control/margins.py

```diff
@@ -10,13 +10,29 @@
 
 import numpy as np
 import scipy as sp
+import matplotlib
+import matplotlib.pyplot as plt
 
 from . import frdata, freqplot, xferfcn
 from .exception import ControlMIMONotImplemented
 from .iosys import issiso
-
-__all__ = ['stability_margins', 'phase_crossover_frequencies', 'margin']
-
+from . import ss
+try:
+    from slycot import ab13md
+except ImportError:
+    ab13md = None
+try:
+    from . import mag2db
+except ImportError:
+    # Likely due the following circular import issue:
+    #
+    # ImportError: cannot import name 'mag2db' from partially initialized module
+    # 'control' (most likely due to a circular import) (control/__init__.py)
+    #
+    def mag2db(mag):
+        return 20*np.log10(mag)
```

Comment:
Try to import from the module where this is defined:

    from .ctrlutil import mag2db